packages/wallet/src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from '@/ui/Link'
+import { Logo } from '@/ui/Logo'
+
+type NavigationItemProps = {
+  name: string
+  href: string
+}
+
+const navigationItems: NavigationItemProps[] = [
+  {
+    name: 'Home',
+    href: '/'
+  },
+  {
+    name: 'Accounts',
+    href: '/accounts'
+  },
+  {
+    name: 'Developer',
```

Comment:
We can name this directly Developer Account, not just Developer, so we know clearly that this is the API keys generation part